src/api/participantsRouter.ts

```diff
@@ -253,7 +253,15 @@ export function createParticipantsRouter() {
         ...users.map((user) =>
           assignClientRoleToUser(kcAdminClient, user.email, 'api-participant-member')
         ),
-        participant!.$query().upsertGraph(data),
+        Participant.query().upsertGraph(
+          {
+            id: participant!.id!,
+            ...data,
+          },
+          {
+            relate: true,
+          }
+        ),
       ];
```

Comment:
Do we also need to send an email for the requestor? Per 
```
Access Requested!
ACCOUNT PENDING
Thank you! We’ve sent your account request for approval. As soon as it’s approved, we’ll send you a confirmation email.
```

Reply:
We have another ticket for this https://atlassian.thetradedesk.com/jira/browse/UID2-1578

Reply:
excellent, ta